hail/python/hailtop/hailctl/batch/wait.py

```diff
@@ -0,0 +1,13 @@
+from .batch_cli_utils import get_batch_if_exists
+
+def init_parser(parser):
+    parser.add_argument('batch_id', type=int)
+
+def main(args, pass_through_args, client):
+    maybe_batch = get_batch_if_exists(client, args.batch_id)
+    if maybe_batch is None:
+        print("Batch with id {} not found".format(args.batch_id))
```

Comment:
prefer f-strings

Reply:
should this exit with a non-zero exit code? I dunno.

Reply:
or maybe raise an error?

Reply:
I think nonzero exit code makes sense

Reply:
Just as an easy way for people using this in a script to check if things went wrong